Classification/cnns/of_cnn_train_val.py

```diff
@@ -118,6 +118,8 @@ def main():
 
     snapshot = Snapshot(args.model_save_dir, args.model_load_dir)
 
+    print(" {} iter per epoch...".format(epoch_size))
```

Comment:
Although it’s a little bit obtrusive to put this line here, I have no more suggestion.